client/src/components/DataFiles/DataFilesBreadcrumbs/DataFilesBreadcrumbs.scss

```diff
@@ -8,3 +11,31 @@
 .breadcrumb-link:hover {
   color: #9d85ef;
 }
+
+/* Breadcrumbs */
+@media only screen and (min-width: 2400px) {
+  .breadcrumbs {
+     max-width: 1000px;
+  }
+}
```

Comment:
@duckonomy Is the reason we don't have `overflow: ellipsis` for the breadcrumbs and title due to collisions with elements on the right hand side?

Reply:
@rstijerina I've added `text-overflow: ellipsis` with the `truncate-with-ellipsis` mixin (for both breadcrumb & title) as suggested by Wes.

Reply:
Ah yes I see. I should have said with respect to the breakpoints. Is the reason we have breakpoints for the breadcurmbs and title due to right-hand side collisions? I was wondering why we had so many, if the overflow would be hidden with ellipsis anyway

Reply:
Yes, I wanted to prevent them from wrapping too readily on resize because they didn't seem to overflow naturally due to them being flex elements with `flex-wrap: wrap` and justified by `space-between`. I was thinking about maybe programmatically resizing their `max-width` by detecting when they wrap but the media queries seemed to be simpler for now (without having to change anything in `SectionHeader`).

Reply:
Got it, thanks!